src/main/java/org/ecommerce/services/impl/UserServiceImpl.java

```diff
@@ -24,35 +25,39 @@ public UserServiceImpl(UserRepository userRepository, PasswordService passwordSe
         this.emailService = emailService;
     }
 
+    public Map<String, String> generateCustomer(Customer customer) {
+        Map<String, String> result = new HashMap<>();
+        if (!emailService.isValidInput(customer.getEmail()))
+            result.put("error", Error.INVALID_EMAIL.getDescription());
+        if (!passwordService.isValidInput(customer.getPassword()))
+            result.put("error", Error.PASSWORD_FORMAT.getDescription());
+        if (result.isEmpty())
+            result.put("generatedCustomer", String.valueOf(save(customer)));
+        return result;
+    }
 
     @Override
     public User save(User entity) {
-        throw new UnsupportedOperationException("Not supported yet.");
```

Comment:
UnsupportedOperationException were ok as method is not implement. Is there any reason to return null/empty results?